compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -142,11 +142,7 @@ impl<'a> From<&'a InterpreterError> for CustomDiagnostic {
                 CustomDiagnostic::simple_error(msg, secondary, location.span)
             }
             InterpreterError::IntegerOutOfRangeForType { value, typ, location } => {
-                let int = match value.try_into_u128() {
-                    Some(int) => int.to_string(),
-                    None => value.to_string(),
-                };
-                let msg = format!("{int} is outside the range of the {typ} type");
+                let msg = format!("{value} is outside the range of the {typ} type");
```

Comment:
Probably not for this PR, but there's a similar error that triggers in another part of the compiler that looks like this:

> The literal `2⁸` cannot fit into `u8` which has range `0..=255`

This is really nice because you immediately know what are the min/max values. We could do the same here too.